examples/train_SB3_gymnasium.py

```diff
@@ -0,0 +1,211 @@
+# stable-baselines3
```

Comment:
Could you create an ipynb version of this example, please?

Reply:
ok I try to find time in the next few days

Reply:
Great, thanks!

Reply:
`train_SB3_gymnasium.ipynb`
https://github.com/AminHP/gym-anytrading/blob/3f7192b3734d399de526f2e0252adbae6ffd6e99/examples/train_SB3_gymnasium.ipynb